api/test/integration/application-flagged-set.e2e-spec.ts

```diff
@@ -60,7 +60,7 @@ describe('Application flagged set Controller Tests', () => {
 
   const createComplexApplication = async (
     emailIndicator: string,
-    nameAndDOBIndicator: string,
+    nameAndDOBIndicator: number,
     listing: string,
     dobIndicator?: string,
```

Comment:
Can we change this to nameIndicator since that's how you are using it?

Reply:
updated!